src/api.js

```diff
@@ -362,8 +362,8 @@
 
     client.call(connect.ClientMethods.CREATE_OUTBOUND_CONTACT, {
       endpoint: connect.assertNotNull(endpoint, 'endpoint'),
-      queueARN: params.queueARN || params.queueId || this.getRoutingProfile().defaultOutboundQueue.queueARN
-    }, {
+      queueARN: (params && (params.queueARN || params.queueId)) || this.getRoutingProfile().defaultOutboundQueue.queueARN
+    }, params && {
```

Comment:
Otherwise, this throws an error: `agent.connect(Endpoint.byPhoneNumber("..."));`